NavigationReact/sample/relay-modern/People.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { createFragmentContainer, graphql } from "react-relay";
+import { NavigationLink, RefreshLink } from "navigation-react";
+
+var People = ({ people, stateNavigator }) => {
+  var rows = people.persons.map(person =>
```

Comment:
Can you use 4 spaces for formatting, please? (Don't bother changing the auto-generated files like `package.json` and `schema.json` - 2 spaces is fine for those).